tasks/i18n.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
```

Comment:
This file is heavily based on this one: https://github.com/Automattic/wp-calypso/blob/master/server/i18n/index.js